packages/affiliates/payouts/dappmining/2021-01-04-0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5.json

```diff
@@ -0,0 +1,22 @@
+{
+  "config": {
+    "empAddress": "0xE4256C47a3b27a969F25de8BEf44eCA5F2552bD5",
+    "startTime": 1609718400000,
+    "endTime": 1610323200000,
+    "totalRewards": 5885,
+    "whitelist": ["0x9a9dcd6b52B45a78CD13b395723c245dAbFbAb71"],
+    "defaultAddress": "0xa0dfF1F09a0DCEda7641b7eB07c3BDbe66a06C35"
+  },
+  "startBlock": {
```

Comment:
Same comment as on the other pr, I think this timeframe should match up to the developer mining window.